deploy/template.yaml

```diff
@@ -0,0 +1,99 @@
+  required: false
+- name: PV_SIZE
+  value: 10Gi
+- name: REPLICAS_COUNT
+  value: "3"
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: assisted-installer
+objects:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: assisted-image-service
+  spec:
+    ports:
+    - port: 8080
+      protocol: TCP
+    selector:
+      service: image-service
+- apiVersion: apps/v1
+  kind: StatefulSet
+  metadata:
+    name: assisted-image-service
+  spec:
+    podManagementPolicy: Parallel
+    replicas: ${{REPLICAS_COUNT}}
+    selector:
+      matchLabels:
+        service: image-service
+    serviceName: assisted-image-service
+    template:
+      metadata:
+        labels:
+          service: image-service
+      spec:
+        containers:
+        - image: ${IMAGE_SERVICE_IMAGE}:${IMAGE_TAG}
+          name: assisted-image-service
+          env:
+          - name: LISTEN_PORT
+            value: "8080"
+          - name: RHCOS_VERSIONS
+            value: ${OS_IMAGES}
+          - name: ASSISTED_SERVICE_SCHEME
+            value: http
+          - name: ASSISTED_SERVICE_HOST
+            value: "assisted-service:8090"
+          - name: DATA_DIR
+            value: "/data"
+          ports:
+          - protocol: TCP
+            containerPort: 8080
+          resources:
+            requests:
+              cpu: 100m
+              memory: 400Mi
+            limits:
+              cpu: 200m
+              memory: 800Mi
+          livenessProbe:
+            httpGet:
+              path: /live
+              port: 8080
+          readinessProbe:
+            httpGet:
+              path: /health
+              port: 8080
+          volumeMounts:
+          - name: assisted-image-service
+            mountPath: "/data"
+        serviceAccountName: assisted-service
+    volumeClaimTemplates:
+    - metadata:
+        name: assisted-image-service
+      spec:
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: ${PV_SIZE}
+- apiVersion: policy/v1
+  kind: PodDisruptionBudget
+  metadata:
+    name: assisted-image-service
+  spec:
+    minAvailable: 2
```

Comment:
Took this right from the assisted service one, do you think we need 2 here also?

Reply:
@osherdp

Reply:
yeah, should do no harm